src/Beman/Example/tests/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+include(GoogleTest)
+
+set(TARGET_LIBRARY_TESTS "${TARGET_LIBRARY}.Test")
+
+add_executable(${TARGET_LIBRARY_TESTS})
```

```suggestion
add_executable(beman.example.tests.identity)
```
Again, using variable names makes the code less obscure.